src/publisher-legacy-tests/pg6/pg6.spec.ts

```diff
@@ -37,7 +37,7 @@ describe("pg@6.x", () => {
         password: "test",
         database: "postgres",
         host: "127.0.0.1",
-        port: 5432,
+        port: 5432
```

Comment:
Hard-coded creds failures...

Reply:
It is for testing purposes so not a blocker, CodeQL will complain each PR so I will fix later